src/scripts/extractAndLaunch

```diff
@@ -23,7 +29,11 @@ while read -r line; do
   if [ -n "$cmd" ]; then
     cmd=$(echo "$cmd" | sed "s/&O/$objlib/g" | sed "s/&N/$objname/g")
     "$SCRIPT_DIR"/launch "$joblog_json" "$cmd" </dev/null
-    # echo "Running $cmd"
+    if [ $? -ne 0 && suppressError -ne "Y"]; then
```

Comment:
@irfanshar  is this the correct place to look for both error conditions?

Reply:
Yes this is